lib/cequel/type.rb

```diff
@@ -282,7 +282,7 @@ def cast(value)
     #
     class Decimal < Base
       def cast(value)
-        value.is_a?(BigDecimal) ? value : BigDecimal.new(value, 0)
+        value.is_a?(BigDecimal) ? value : BigDecimal(value, 0)
```

Comment:
Do we even need the ternary? Most of the `<class>()` methods handle being given a value of that target type gracefully.

Reply:
Honestly I don't know how `BigDecimal()` works inside, and probably shouldn't. Maybe it's idempotent. Maybe it doesn't do extra work in this case. Maybe it will work like this forever. Too much assumptions for me :-)
On the other hand I find current code straightforward and I understand decision making process behind it.